src/main/java/com/google/identitytoolkit/GitkitClient.java

```diff
@@ -111,6 +112,29 @@ public static GitkitClient createFromJson(String configPath) throws JSONExceptio
          .build();
   }
 
+    /**
+     * Constructs a Gitkit client from a JSON config file
+     *
+     * @param configPath Path to JSON configuration file
+     * @return Gitkit client
+     */
+    public static GitkitClient createFromJson(String configPath, Proxy proxy) throws JSONException, IOException {
```

Comment:
The existing createFromJson(String configPath) method can now be implemented by createFromJson(configPath, null);